packages/libs/components/src/stories/plots/BipartiteNetwork.stories.tsx

```diff
@@ -193,7 +193,6 @@ function genBipartiteNetwork(
   return {
     nodes,
     links,
-    column1NodeIDs,
-    column2NodeIDs,
+    partitions: [{ nodeIds: column1NodeIDs }, { nodeIds: column2NodeIDs }],
```

Comment:
For sake of completeness, I think it'd be good to change "columnXblah" to "partitionXblah" throughout this file as well.